chips/nrf5x/src/timer.rs

```diff
@@ -265,10 +265,10 @@ pub struct TimerAlarm {
 
 // CC0 is used for capture
 // CC1 is used for compare/interrupts
-const ALARM_CAPTURE: usize = 0;
-const ALARM_COMPARE: usize = 1;
-const ALARM_INTERRUPT_BIT: registers::Field<u32, Inte::Register> = Inte::COMPARE1;
-const ALARM_INTERRUPT_BIT_SET: registers::FieldValue<u32, Inte::Register> = Inte::COMPARE1::SET;
+const ALARM_CAPTURE: usize = 1;
+const ALARM_COMPARE: usize = 0;
```

Comment:
Something is weird here... does that comment need to update, should these constants not have changed?

Reply:
I don't know. @dcellucci?

Reply:
I'm nearly 100% sure it doesn't matter which is which and the change should either be reverted or the comment should change (and it doesn't matter which one)

Reply:
✅